packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -589,12 +560,7 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
     // RectangularSliderValueIndicatorShape is used. In all other cases, the
     // value indicator is assumed to be the same as the active color.
     final RangeSliderValueIndicatorShape valueIndicatorShape = sliderTheme.rangeValueIndicatorShape ?? _defaultValueIndicatorShape;
-    Color valueIndicatorColor;
-    if (valueIndicatorShape is RectangularRangeSliderValueIndicatorShape) {
```

Comment:
I think this should stay, since we still do want the same color assignments for the old indicator shape.

Reply:
Done